Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1540,12 +1540,17 @@ def search_file_command(file):
     return command_results
 
 
-def get_tags_for_generic_conext(tags: dict):
-    keys = ['TagGroups', 'Aliases', 'PublicTagName', 'TagName']
-    sub_keys = ['TagGroupName']
-    generic_context_tags = {key: tags[0].get(key) for key in keys}
-    generic_context_tags['tagGroups'] = {key: tags[0].get(key) for key in sub_keys}
-    return remove_empty_elements(generic_context_tags)
+def get_tags_for_generic_context(tags: Optional[dict]):
```

Comment:
So is tags a list or a dict?

Reply:
At the examples I tested, Tags was always a one item list, at the start I (wrongly) had in mind that this is always the case, hence used the direct indexing of 0. Since I realised it can be more than one, I added the loop.

Reply:
Got it, so please change the type here to Optional[list]